homework/english/Anastasia_Sheremet/Lecture1/Task_2.md

```diff
@@ -0,0 +1,60 @@
+Домашнее задание по лексике
+Напишите по 1 предложению к каждому слову из лексического блока в 1 уроке (слова есть в презентации).
+
+to boot
+He is booting updates to the server now.
+
+to create
+At the previous lesson we learned how to create a new class.
+
+to connect
+He tryed to connect to the server but every time he received 500 mistake.
+
+to debug
+It took him 20 minutes to debug programm.
```

Comment:
To debug the program - but here “the” should be presented ,because you have mentioned exactly what the program